wordcloud/wordcloud_cli.py

```diff
@@ -0,0 +1,104 @@
+        help='specify file of stopwords (containing one word per line) to remove from the given text after parsing')
+    parser.add_argument('--imagefile', metavar='file', type=argparse.FileType('w'),
+        help='file the completed PNG image should be written to')
+    parser.add_argument('--fontfile', metavar='path', dest='font_path',
+        help='path to font file you wish to use - defaults to using DroidSansMono')
+    parser.add_argument('--mask', metavar='file', type=argparse.FileType(),
+        help='mask to use for the image form')
+    parser.add_argument('--colormask', metavar='file', type=argparse.FileType(),
+        help='color mask to use for image coloring')
+    parser.add_argument('--relative_scaling', type=float, default=0,
+        metavar='rs', help='optional scaling of words by frequency (0 - 1)')
+    parser.add_argument('--scale', type=int, default=1,
+        metavar='scale', help='optional scaling the default image size')
+    parser.add_argument('--margin', type=int, default=2,
+        metavar='width', help='spacing to leave around words')
+    parser.add_argument('--width', type=int, default=400,
+        metavar='width', help='optional output image width')
+    parser.add_argument('--height', type=int, default=200,
+        metavar='height', help='optional output image height')
+    parser.add_argument('--color', metavar='color',
+        help='use given color as coloring for the image - accepts any value from PIL.ImageColor.getcolor')
+    parser.add_argument('--background', metavar='color', default='black',
+        help='use given color as background color for the image - accepts any value from PIL.ImageColor.getcolor')
+    args = parser.parse_args()
+
+    if sys.stdin.isatty() and args.words is None:
+        raise ValueError('need to specify a word file if not using stdin')
+
+    words = args.words or sys.stdin
+    imagefile = args.imagefile or sys.stdout
+    with words:
+        text = words.read()
+
+    if args.stopwords:
+        with args.stopwords:
+            stopwords = set(map(str.strip, args.stopwords.readlines()))
+    else:
+        stopwords = None
+
+    if args.mask:
+        mask = np.array(Image.open(args.mask))
+    else:
+        mask = None
+
+    color_func = wc.random_color_func
+    if args.colormask:
+        image = np.array(Image.open(args.colormask))
+        color_func = color_func_from_image(image)
+
+    if args.color:
+        color_func = get_single_color_func(args.color)
+
+    wordcloud = wc.WordCloud(stopwords=stopwords, mask=mask,
+        width=args.width, height=args.height, font_path=args.font_path,
+        scale=args.scale, margin=args.margin, relative_scaling=args.relative_scaling,
+        color_func=color_func, background_color=args.background).generate(text)
+    image = wordcloud.to_image()
+
+    b = io.BytesIO()
+    image.save(b, format='png')
+    with imagefile:
+        imagefile.write(b.getvalue())
+
+if __name__ == '__main__':
+    main()
```

Comment:
I think you should add ``args`` as a parameter for ``main``, which would allow testing.